plugins/ingest/src/main/java/org/elasticsearch/plugin/ingest/IngestModule.java

```diff
@@ -40,6 +41,7 @@ protected void configure() {
         binder().bind(PipelineStoreClient.class).asEagerSingleton();
 
         registerProcessor(SimpleProcessor.TYPE, SimpleProcessor.Builder.Factory.class);
+        registerProcessor(GrokProcessor.TYPE, GrokProcessor.Builder.Factory.class);
```

Comment:
really I think we should not register class instances but rather instance of a the factory or rather use a simple interface we define ourself? like look at how IndexModule works now